index.html

```diff
@@ -0,0 +1,271 @@
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>   
+  </header>
+    <section class="headline" id="headline">
+      <div class="main-box">
+        <h1 class="main">
+        Hi There.<br>
+        I'm Tessy
+        </h1>
+        <sub-title class="sub">
+        <h3>I'am a software developer</h3>
+        </sub-title>
+        <p class="p">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      </div>
+      <div class="mobile-img"><img src="./images/illustration-header-mobile.png" alt="Diagram illustration of works" class="headline-img"></div>       
+    </section>
+    <ul class="social first-icon">
+      <li class="git fico">
+        <a href="#"><img src="./images/Icon -GitHub.png" alt="GitHub logo"></a>
+      </li>
+      <li class="lin fico"><a href="#"><img src="./images/Icon - Linkedin.png" alt="Linkedin logo"></a></li>
+      <li class="ter fico"><a href="#"><img src="./images/Icon - Angelist.png" alt="Angelist logo"></a></li>
+      <li class="twitter fico"><a href="#"><img src="./images/Icon - Twitter.png" alt="Twitter logo"></a></li>
+      <li class="mess fico"><a href="#"><img src="./images/Icon - Medium.png" alt=" M logo"></a></li>
+    </ul> 
+    <!-- <div class="desktop-img"><img src="./images/Illustration-header@2x.png" alt="Diagram illustration 2"></div> -->
+</main>
+
+<section class="recent">
+  <div class="title">
+    <h2>My Recent works</h2>
+    <div class="v"></div>
+  </div>
+  <div class="frame" id="frame">
+  <div class="featured">
+    <div class="imgp"></div>
+    <!-- <div class="imp-desk"><img src="./images/ImgPlaceholder (2).png" alt=" Diagram illustration 3"></div> -->
+    <div class="block">
+      <h2 class="post">Multi-Post Stories</h2>
+      <p class="support">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <ul class="tag-main">
+        <li class="tag1"><a href="#">css</a></li>
+        <li class="tag2"><a href="#">html</a></li>
+        <li class="tag3"><a href="#">bootstrap</a></li>
+        <li class="tag4"><a href="#">Ruby</a></li>
+      </ul>
+      <section class="buttom">
+        <a href="#" class="see" id="see">See project</a>
+      </section>
+    </div>
+  </div>
+  <section class="frame-sub" id="frame-sub">
+    <div class="card-works card1">
+      <div class="mask">
+        <div class="img-placeholder a">
+          <div class="div">
+            <div class="right-works">
+              <h2 class="pro">Profesional Art Printing Data</h2>
+              <h6 class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6>
```

Comment:
- [ ] You have applied the change previous reviewer adressed but only for one card at the top. Kindly apply this change to all cards by adding p tag instead h6.